homeassistant/components/websocket_api/decorators.py

```diff
@@ -117,11 +115,29 @@ def websocket_command(
     """Tag a function as a websocket command."""
     command = schema["type"]
 
-    def decorate(func):
+    def decorate(func: const.WebSocketCommandHandler) -> const.WebSocketCommandHandler:
         """Decorate ws command function."""
         # pylint: disable=protected-access
-        func._ws_schema = messages.BASE_COMMAND_MESSAGE_SCHEMA.extend(schema)
-        func._ws_command = command
+        func._ws_schema = messages.BASE_COMMAND_MESSAGE_SCHEMA.extend(schema)  # type: ignore[attr-defined]
+        func._ws_command = command  # type: ignore[attr-defined]
+        return func
+
+    return decorate
+
+
+def async_websocket_command(
```

Comment:
Since `async_response` uses `@functools.wraps`, it should copy over all stored data on the function (`_ws_schema`). So we should be able to just turn around the order of the decorators and things will work with existing decorator

Reply:
You're right, thanks for the hint! So I guess this is not very useful then.